bundles/action/org.openhab.action.xmpp/README.md

```diff
@@ -18,7 +18,7 @@ The XMPP Action service can be configured in `services/xmpp.cfg`.
 | servername   |               |          | The XMPP service to use, e.g. `jabber.de`. A list of public XMPP services can be found at https://xmpp.net/directory.php                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | securitymode | `disabled`    |       No | The Security mode used for the XMPP connection. Can be either `required` or `disabled`. Defaults to `disabled`, which means that TLS will not be used.  Warning: If you change this to non-disabled, then you must make sure that your TLS server certificate can be validated, otherwhise the connection will fail.                                                                                                                                                                                                                                                                          |
-| tlspin       |               |          | The TLS Pin used to verify the XMPP service's certificate. Set this in case openHAB's default `SSLContext` is unable to verfiy it (e.g. because the XMPP service uses a self-signed certificate). The PIN value is bascially the hash of the certificate in hexadecimal notation. You have to set `securitymode` to `required` to enable TLS for XMPP connections. For information on how to generate the PIN visit https://github.com/Flowdalic/java-pinning .  Example: `tlspin=CERTSHA256:83:F9:17:1E:06:A3:13:11:88:89:F7:D7:93:02:BD:1B:7A:20:42:EE:0C:FD:02:9A:BF:8D:D0:6F:FA:6C:D9:D3` |
+| tlspin       |               |          | The TLS Pin used to verify the XMPP service's certificate. Set this in case openHAB's default `SSLContext` is unable to verfiy it (e.g. because the XMPP service uses a self-signed certificate). The PIN value is bascially the hash of the certificate in hexadecimal notation. You have to set `securitymode` to `required` to enable TLS for XMPP connections. For information on how to generate the PIN visit https://github.com/Flowdalic/java-pinning or see an example at the bottom of this page. Example: `tlspin=CERTSHA256:83:F9:17:1E:06:A3:13:11:88:89:F7:D7:93:02:BD:1B:7A:20:42:EE:0C:FD:02:9A:BF:8D:D0:6F:FA:6C:D9:D3` |

```

Comment:
Please correct the spellings of `verfiy` and `bascially`.
Please link to the example instead of just mentioning it.

Reply:
old spellings corrected.

I find it confusing to link to an example which is just centimeters further, but sure. While doing that I'll also fix the link to the external site.